xwiki-platform-core/xwiki-platform-help/xwiki-platform-help-ui/src/main/resources/XWiki/XWikiSyntaxMacrosList.xml

```diff
@@ -56,7 +55,28 @@
     #set ($displayId = $descriptor.id)
     #set ($displayVisibility = "Global")
   #end
-  |$displayId|$descriptor.name|$!descriptor.defaultCategory|$descriptor.description|$displayVisibility
+  |$displayId|##
+  #if($services.localization.get("rendering.macro.${descriptor.id.id}.name"))
+    $services.localization.render("rendering.macro.${descriptor.id.id}.name")##
```

Comment:
Indeed we could probably make the economy of one call, even though I doubt the perf improvement would be perceptible. I think the logic can be generalized and I'll introduce a macro to minimize the code duplication.
 I'm hesitating to introduce an API for the translations as it is probably the single place where it will be used. The CKEditor implementation will be based on javascript translations and will not call that new API.